qm-plugins/qm-cron/class-qm-cron-collector.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+class QM_Cron_Collector extends \QM_Collector {
```

Comment:
Here and below: does it make sense to use a fully qualified class name? As far as I can see, we don't have a namespace here.

```suggestion
class QM_Cron_Collector extends QM_Collector {
```